pages.id/linux/add-apt-repository.md

```diff
@@ -0,0 +1,20 @@
+# add-apt-repository
+
+> Mengelola definisi repositori `apt`.
```

Comment:
```suggestion
> Kelola definisi repositori `apt`.
```

Can you please change this (and subsequent) sentences to use imperative mood (see [this](https://share.reinhart1010.id/tldrguidelinesidspecific) and [this](https://share.reinhart1010.id/tldridimperativemood))?